trick-2.php

```diff
@@ -2,15 +2,15 @@
 
 /* Insert Magic Code Here */
 
-// NO CHANGES ALLOWED BELOW
+$аrray = [0,0,0]; $arraу = [3,3,3];
 
-var_dump(count($array), array_sum($array));
+// NO CHANGES ALLOWED BELOW
 
+var_dump(count($аrray), array_sum($аrray));
```

Comment:
This violates the no changes allow below statement.

But cool idea to substitute a glyph with another very similar glyph.